admin_manual/installation/harden_server.rst

```diff
@@ -297,6 +297,26 @@ Setup fail2ban
 Exposing your server to the internet will inevitably lead to the exposure of the 
 services running on the internet-exposed ports to brute force login attempts.
 
+This guide will enable blocking of the originating IP addresses at an operating
+system level, so the webserver, PHP and the database do not need to handle this
+unnecessary traffic at all.
+
+Nextcloud prerequisites
+^^^^^^^^^^^^^^^^^^^^^^^
+
+Nextcloud logs failed login attempts in ``nextcloud.log`` with log level ``2``,
+so you need to define a ``loglevel`` of ``2`` or less in ``config.php``.
+
+Make sure your ``nextcloud.log`` is writeable by your webserver user, possibly by
```

Comment:
In theory the nextcloud.log is owned by the webserver user anyway, as it's the one creating and writing it?

Reply:
Only in theory :-)
Mine is not, as the whole installation is owned by a specific linux user as I never let the `www-data` user own its own executables (r/w), as that opens up the possibility for exploits to change the application's code.
Most admins I know do it this way.